gh-pages-stub/public/js/scripts.js

```diff
@@ -102,34 +107,35 @@ var COLLAPSING_CODE_BLOCS = (function () {
 
   function init() {
 
-    function formatHTML(smallText, fullText, id, moreButtonText) {
+    function formatHTML(smallText, fullText, id, MORE_TEXT) {
       return smallText +'<div id="id' + id + '" class="' + COLLAPSE_CLASS + '">' +
       fullText + '</div>' + '<br><button data-toggle="' + COLLAPSE_CLASS + '" data-target="#id' +
-      id + '" class="' + COLLAPSE_BUTTON_CLASS + ' ' + MORE_CLASS + '">' + moreButtonText + '</button>';
+      id + '" class="' + COLLAPSE_BUTTON_CLASS + ' ' + MORE_CLASS + '">' + MORE_TEXT + '</button>';
     }
 
-    function attachCollapseHandler(moreButtonText, lessButtonText) {
+    function attachCollapseHandler() {
       $(COLLAPSE_BUTTON_SELECTOR).click(function(){
-         if( $(this).hasClass(MORE_CLASS)) {
-             $(this).removeClass(MORE_CLASS);
-             $(this).html(lessButtonText);
-         } else {
-             $(this).addClass(MORE_CLASS);
-             $(this).html(moreButtonText);
-         }
+          if($(this).hasClass(MORE_CLASS)) {
+            $(this).removeClass(MORE_CLASS);
+            $(this).html(LESS_TEXT);
+          } else {
+            $(this).addClass(MORE_CLASS);
+            $(this).html(MORE_TEXT);
+          }
       });
     }
 
-    function processAllCodeBlocs(linesToShow, moreButtonText, lessButtonText) {
+    function processAllCodeBlocs() {
       $(CODE_BLOCK_SELECTOR).each(function(index) {
         var content = UTILS.splitByLines($(this).html());
-        if(content.length > linesToShow) {
-          var SMALL_TEXT = content.slice(0, linesToShow).join("\n");
-          var FULL_TEXT = content.slice(linesToShow).join("\n");
-          $(this).html(formatHTML(SMALL_TEXT, FULL_TEXT, index, moreButtonText));
+        var numberOfHiddenLines = content.length - NUMBER_OF_LINES_TO_SHOW;
+        if(content.length > NUMBER_OF_LINES_TO_SHOW && numberOfHiddenLines > MIN_NUMBER_OF_LINES_TO_HIDE) {
```

Comment:
if(content.length > NUMBER_OF_LINES_TO_SHOW + MIN_NUMBER_OF_LINES_TO_HIDE)
would be sufficient here: you don't seem to need numberOfHiddenLines